stream-api/src/main/kotlin/fund/cyber/markets/api/trades/TokensPairTradesBroadcaster.kt

```diff
@@ -48,12 +50,12 @@ class TokensPairTradesBroadcaster(
     fun registerChannel(channel: WebSocketChannel) {
 
         launch(applicationSingleThreadContext) {
-            val lastTradesAsJson = jsonSerializer.writeValueAsString(lastTrades.elements)
-            WebSockets.sendText(lastTradesAsJson, channel, null)
             registeredChannels.add(channel)
         }
     }
 
+    fun getRandomTradeFromBroadcaster() : Trade = lastTrades.getElement(Int.rand(0, lastTrades.elements.size))
```

Comment:
Not sure on this, but `Int.rand` looks little bit non native for Kotlin. Maybe replace with top-level `randInt` function? Or add extension to `IntRange` class?

```
fun IntRange.random() = (Math.random() * (this.start - last) + last).toInt()

(1..collection.size).random()
```